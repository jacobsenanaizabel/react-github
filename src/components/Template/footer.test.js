import React from 'react'
import { shallow } from 'enzyme'
import '../../setupTests'

import Footer from './footer'

describe('<Footer />', () => { 
    it('renders footer', () => {
        shallow(<Footer />);
    })

    it('render the title', () => {
        const wrapper = shallow(<Footer/>);
        expect(wrapper.text()).toContain('Feito com ‚ù§ por Ana Jacobsen.')
    })

});