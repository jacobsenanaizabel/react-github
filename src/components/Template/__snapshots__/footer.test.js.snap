// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders footer snapshot 1`] = `
<MemoryRouter
  keyLength={0}
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Component>
      <footer
        class="footer"
      >
        <div
          class="container"
        >
          <span>
            Feito com ‚ù§ por Ana Jacobsen.
          </span>
        </div>
      </footer>
    </Component>
  </Router>
</MemoryRouter>
`;
